### ISSUE TITLE
the website  '0xproject.localhost:3572'  refuse to connect

### ISSUE THREAD

**Issue by zhouzhou84** [0xProject/0x-monorepo#968]
### Description
I follow the `https://github.com/0xProject/0x-monorepo/blob/v2-prototype/packages/website/README.md` to do and at first I change yarn to 1.6.0.
but when I type `PKG=@0xproject/website yarn watch` , it shows:
```
[11:19:54] Starting compilation in watch mode...
 | 
 | [11:19:56] Found 0 errors. Watching for file changes.
 | 

```
but I visit 0xproject.localhost:3572 in my browser, it shows:`ERR_CONNECTION_REFUSED` ,
what's wrong with it?

**Comment by fabioberger** [comment]
@zhouzhou84 CD into the `website` directory and run `yarn run watch_without_deps`.

**Comment by zhouzhou84** [comment]
@fabioberger I'm sorry, yesterday I run the `PKG=@0xproject/website yarn build` in the `~/0x-monorepo/packages/website` folder, BUT the tutorial says:"To build this package and all other monorepo packages that it depends on, run the following from the monorepo root directory:"  I need CD into monorepo root directory like `monorepo`, so I do it ,however the use of my memory increase fast, at last it break down, and I check the output in terminal, it says:`@0xproject/website
$ NODE_ENV=production node --max_old_space_size=8192 ../../node_modules/.bin/webpack; exit 0;
`, what's the matter?? bro

**Comment by zhouzhou84** [comment]
@fabioberger hello, I describe the step I followed:
git the repository;
add the line `127.0.0.1 0xproject.localhost` to `/etc/hosts`
CD into the folder:`~/0xmonorepo/packages/website` to run `yarn install`; it's ok;
CD into the folder:`~/0xmonorepo` to run `PKG=@0xproject/website yarn build`; it's ok;
and in the `~/0xmonorepo` folder to run ` PKG=@0xproject/website yarn watch`; it's ok;
Visit 0xproject.localhost:3572 ,but it return connect refused.
is there anything wrong?

**Comment by BMillman19** [comment]
hello @zhouzhou84 there's an outstanding PR to fix some build issues with website: https://github.com/0xProject/0x-monorepo/pull/988

There is still a problem with the watch command that we're ironing out but the website should be buildable with the following commands after this PR gets merged:

```bash
cd ~/0x-monorepo
yarn install && PKG=@0xproject/website yarn build
cd packages/website && yarn watch_without_deps
```

**Comment by zhouzhou84** [comment]
Ok,I have a try On 2018-08-21 04:42 , Brandon Millman Wrote: hello @zhouzhou84 there's an outstanding PR to fix some build issues with website: #988 There is still a problem with the watch command that we're ironing out but the website should be buildable with the following commands after this PR gets merged: cd ~/0x-monorepo
yarn install && PKG=@0xproject/website yarn build
cd packages/website && yarn watch_without_deps — You are receiving this because you were mentioned. Reply to this email directly, view it on GitHub, or mute the thread.
